docs/WritingTests.asciidoc

```diff
@@ -1065,3 +963,140 @@ the SUSE distribution's serial terminal utility library). However some
 programs ignore this, but piping there output into +tee+ is usually enough to
 stop them outputting non-printable characters.
 
+
+== Test Development tricks
+=== Modifying setting of an existing test
+
+There is no interface to modify existing tests but the clone_job.pl script
+can be used to create a new job that adds, removes or changes
+settings. This script is located at +/usr/share/openqa/script/+.
+
+[source,sh]
+--------------------------------------------------------------------------------
+/usr/share/openqa/script/clone_job.pl --from localhost --host localhost 42 FOO=bar BAZ=
+--------------------------------------------------------------------------------
+
+If you do not want a cloned job to start up in the same job group as the job
+you cloned from, e.g. to not pollute build results you the job group can be
+overwritten, too, using the special variable +_GROUP+. Add the quoted group
+name, e.g.:
+
+[source,sh]
+-------------
+clone_job.pl --from localhost 42 _GROUP="openSUSE Tumbleweed"
+-------------
+
+The special group value +0+ means that the group connection will be separated
+and the job will not appear as a job in any job group, e.g.:
+
+[source,sh]
+-------------
+clone_job.pl --from localhost 42 _GROUP=0
+-------------
+
+==== Backend variables for faster test execution
```

Comment:
@coolo as suggested in https://github.com/os-autoinst/os-autoinst/pull/938#issuecomment-378210897